lib/send_messages/send_messages_request.js

```diff
@@ -1,4 +1,4 @@
-module.exports = function(userId, password, content, recipients, callback) {
+module.exports = function(userId, password, content, recipients, deliveryReport, scheduledTime, messageId, callback) {
```

Comment:
How does this function work if they want to provide just userid, pwd, content, recip, and callback? do they have to put nulls in between for the other params?

If they do then this is a breaking change and we cant merge it.